rasa/core/actions/action.py

```diff
@@ -170,7 +170,7 @@ def action_from_name(
     ):
         return ActionRetrieveResponse(name)
 
-    if name.startswith(UTTER_PREFIX):
+    if name.startswith(UTTER_PREFIX) or name in domain.action_names:
```

Comment:
Won't this prevent remote actions?

Reply:
omg, yes of course. That was a bling auto complete 🙈